app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -7,16 +7,16 @@ class PermissionTemplateForm
       self.model_class = PermissionTemplate
       self.terms = []
       delegate :access_grants, :access_grants_attributes=, :release_date, :release_period, :visibility, to: :model
-      delegate :available_workflows, :active_workflow, :source_model, to: :model
+      delegate :available_workflows, :active_workflow, :source, to: :model
```

Comment:
`:source_model` is still available.  It is defined as a method to allow for a deprecation message.